addons/sale/models/product_template.py

```diff
@@ -19,6 +19,22 @@ class ProductTemplate(models.Model):
         help="Manually set quantities on order: Invoice based on the manually entered quantity, without creating an analytic account.\n"
              "Timesheets on contract: Invoice based on the tracked hours on the related timesheet.\n"
              "Create a task and track hours: Create a task on the sales order validation and track the work hours.")
+    service_tracking = fields.Selection(
+        selection=[
+            ('no', 'Nothing'),
+        ],
+        string="Link to Service",
+        default="no",
+        compute="_compute_service_tracking",
+        store=True,
+        readonly=False,
+        help="Link this product to different type of service. \
+        For project/task related Types: \
+        On Sales order confirmation, this product can generate a project and/or task. \
+        From those, you can track the service you are selling.\n \
+        'In sale order\'s project': Will use the sale order\'s configured project if defined or fallback to \
+        creating a new project based on the selected template.\
```

Comment:
This help must be adapted.
Who decided the string of the field? We won't use `Create on Order` anymore ?

Reply:
Reverted string!
Waiting for help string from @dalaOdoo